platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -53,20 +53,22 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ConfigurationMigrationManager.class);
 
-  private static final String PRODUCT_VERSION_FILENAME = "Version.txt";
+  private static final String PRODUCT_BRANDING_FILENAME = "Branding.txt";
 
-  private static final String EXPORT_EXTENSION = ".zip";
+  private static final String PRODUCT_VERSION_FILENAME = "Version.txt";
 
-  private static final String EXPORTED = "exported";
+  private static final String EXPORT_EXTENSION = ".dar";
```

Comment:
So I noticed that we are in essence using an implicit contract for the export extension (since we had to manually change/put ".dar" throughout all these files). Is this something that could be shared such that changes in the future will not need manual updates? Or is this something that will likely never change so just relying on the assumption that the export extension will be .dar is good enough?

Reply:
I don't think it will change anymore. It was zip before but with the encryption ticket Mike is working we had to come up with our own which became .dar (DDF archive). I think we are set now.